src/nidm/experiment/Acquisition.py

```diff
@@ -83,10 +83,7 @@ def acquisition_object_exists(self, uuid):
         :param uuid: full uuid of acquisition
         :return: True if exists, False otherwise
         """
-        if uuid in self._acquisition_objects:
-            return True
-        else:
-            return False
+        return bool(uuid in self._acquisition_objects)
```

Comment:
me is curious:

in principle even `bool` isn't needed, since `in` would return bool ,right?

```suggestion
        return uuid in self._acquisition_objects
```
or that is really to guarantee it if somehow `self._acquisition_objects` overloads some operator and returns smth else?

Reply:
I was just following what Pylint suggested, though given how mypy is with the return type of `==` on arbitrary types, I wouldn't be surprised if it complained about the lack of `bool()` here as well.